apps/storybook/src/index.css

```diff
@@ -1,68 +1,68 @@
 :root {
```

Comment:
Changes in this file seem accidental since there are no changes in Prettier / Eslint

Reply:
see above